collect_app/src/main/java/org/odk/collect/android/activities/FormMapActivity.java

```diff
@@ -66,18 +64,19 @@
 /**
  * Show a map with points representing saved instances of the selected form.
  */
-public class FormMapActivity extends CollectAbstractActivity {
+public class FormMapActivity extends SelectionMapActivity {
```

Comment:
You converted some classes to kotlin maybe it's good time to do the same with this class too?

Reply:
Good point. Best to leave it for the moment though as I'll be moving most of the code from `FormMapActivity` over to `SelectionMapActivity` and it will be simpler to deal with the conversion piece by piece I think.

Reply:
Ok if it's more convenient for you.